en/django_forms/README.md

```diff
@@ -44,36 +44,48 @@ So once again we will create a link to the page, a URL, a view and a template.
 
 ## Link to a page with the form
 
-It's time to open `blog/templates/blog/base.html` in the code editor. In the `div` named `page-header`, we will add a link:
+Before we add the link, we need some icons to use as buttons for the link. For this tutorial, download [file-earmark-plus.svg](https://raw.githubusercontent.com/twbs/icons/main/icons/file-earmark-plus.svg) and save it in the folder `blog/templates/blog/icons/`
+
+> Note: To download the svg image, click on the link and save the file using the keyboard shortcut `Ctrl + S`(Windows/Linux) or `Cmd + S`(MacOS).
```

Comment:
Another (maybe better?) option: Open the context menu on the link (usually by right-click on it) and choose "Save as ..."

Reply:
@das-g I was thinking same thing. But then I remember users use different browsers and they have different content menu. For example, firefox show "Save link as". So if we use context menu, do we have to cover all browsers's context menu option to save?

Reply:
@das-g I tested the save from link option on the latest stable distributions of Firefox, Chrome, Edge, Safari, and IE 11. Found "Save link as" for all the browsers except Internet Explorer 11. Internet Explorer 11 has "Save target as" instead.

So should I change the note as follows?

> Note: To download the SVG image,  open the context menu on the link (usually by right-clicking on it) and choose "Save link as" option. Now you can navigate to the desired directory in the window on your screen and save the file.

Reply:
Yes, that seems like a more natural way to save the file.

Funny that IE's menu entry label is actually more accurate, as it _is_ the link _target_ that's being saved (the file the link points at), and indeed not the link itself. :upside_down_face: (Off course, our description should use the more common "Save link as")

Reply:
Resolved in https://github.com/DjangoGirls/tutorial/pull/1681/commits/27483eb7bf7703eb6c02e51aaaf9702ceae631b8